ansible/roles/infra-azure-template-destroy/tasks/main.yml

```diff
@@ -39,7 +39,7 @@
           fail:
             msg: "FAIL {{ az_resource_group }} Azure delete group"
           when:
-            - not azure_result|succeeded
+            - not azure_result is succeeded
```

Comment:
is 
```
not azure_result is succeeded
```
correct for all versions ?
Or
```
 azure_result is not succeeded
```

I just tried:

```yaml
- hosts: localhost
  gather_facts: no
  connection: local
  tasks:
          - shell: exit 2
            register: r
            ignore_errors: yes

          - debug:
                  msg: OK not is
            when: not r is succeeded

          - debug:
                  msg: OK is not
            when: r is not succeeded
```

with 2.7.7 and both work, no complain, so i guess we are fine?

`is failed` is probably more elegant though :smile: